pynest/examples/balancedneuron.py

```diff
@@ -123,13 +123,13 @@
 
 
 def output_rate(guess):
-    print("Inhibitory rate estimate: %5.2f Hz" % guess)
+    print("Inhibitory rate estimate: %5.2f spks/s" % guess)
```

Comment:
Please update to use Python format strings here and in other places touched by this PR.

Reply:
Done!